daemons/attrd/attrd_messages.c

```diff
@@ -119,19 +119,19 @@ handle_confirm_request(pcmk__request_t *request)
 
         pcmk__set_result(&request->result, CRM_EX_OK, PCMK_EXEC_DONE, NULL);
         return NULL;
-    } else {
-        return handle_unknown_request(request);
     }
+
+    return handle_unknown_request(request);
```

Comment:
We could have less total indentation, both here and in `handle_remove_request`, by making this the return-early case. I'm fine with using your approach for consistency though, if we want to stick with this as a standard format.